themes/default/content/docs/clouds/aws/guides/eks.md

```diff
@@ -1311,7 +1311,7 @@ assume_role_policy = json.dumps({
     "Statement": [{
         "Action": "sts:AssumeRole",
         "Effect": "Allow",
-        "Sid": None,
+        "Sid": "",
```

Comment:
@omidraha Since this property is apparently [optional](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_sid.html), what do you think about omitting it altogether? When I do that, it looks like the policy gets created successfully (and without a `Sid`) -- and when I then change the `Sid` to `""`, Pulumi detects no changes.

```suggestion
```

Reply:
In the tests related to the library, an empty string has been used as a test case.

You can find an example of this usage at the following link:

https://github.com/pulumi/pulumi-aws/blob/c0dcccb3d7aef5bd661e2f74ec4c5361273d7c12/sdk/python/pulumi_aws/iam/role_policy.py#L191

May be it have been preferred to explicitly include an empty string for clarity in the policy structure.

And also, the record created in the Amazon console will be different.

- ![3079_01](https://github.com/pulumi/pulumi-hugo/assets/397672/6a759dbe-c81f-4d60-83f7-94796423ba66)
- ![3079_02](https://github.com/pulumi/pulumi-hugo/assets/397672/adfcde7c-fe4c-47aa-b168-d05130a5688c)